projects/aca-playwright-shared/src/page-objects/components/dataTable/data-table.component.ts

```diff
@@ -281,4 +289,41 @@ export class DataTableComponent extends BaseComponent {
   async getRowAllInnerTexts(name: string): Promise<string> {
     return (await this.getRowByName(name).locator('span').allInnerTexts()).toString();
   }
+
+  async getFirstElementDetail(name: string): Promise<string> {
+    const firstNode = this.getColumnValuesByName(name).first();
+    return firstNode.innerText();
+  }
+
+  async isEmpty(): Promise<boolean> {
+    return this.emptyList.isVisible();
+  }
+
+  async getEmptyStateTitle(): Promise<string> {
+    const isEmpty = await this.isEmpty();
+    if (isEmpty) {
+      return this.emptyListTitle.innerText();
+    }
+    return '';
```

Comment:
It can be shorter like: return await this.isEmpty() ? this.emptyListTitle.innerText() : '';

Reply:
added suggested  changes. looks good